guides/v2.2/magento-functional-testing-framework/release-2/test/actions.md

```diff
@@ -729,6 +729,19 @@ Attribute|Type|Use|Description
 `before`|string|optional| `stepKey` of action that must be executed next.
 `after`|string|optional| `stepKey` of preceding action.
 
+### generateDate
+
+Generates a date for use in `{$stepKey}` format in other test actions.
+
+Attribute|Type|Use|Description
+---|---|---|---
+`date`|string|required| Data input to parse, uses same functionality as php `strtotime()` function.
```

Comment:
Date input?  
parse. Uses the same functionality as the PHP

Reply:
🆗

Reply:
🆗